app/Console/Commands/UserCreate.php

```diff
@@ -71,7 +71,7 @@ public function handle()
 
         $email = $this->ask('Email');
 
-        if(User::whereEmail($email)->exists()) {
+        if(User::whereRaw('lower(email) = ?', [strtolower($email)])->exists()) {
```

Comment:
I would keep this as it was and have this change on line 72 as per my next comment.
```suggestion
        if(User::whereEmail($email)->exists()) {
```